package.json

```diff
@@ -73,7 +73,7 @@
     "css-loader": "^3.0.0",
     "devtron": "^1.2.1",
     "docx": "^5.0.2",
-    "electron": "^10.2.0",
+    "electron": "^10.4.0",
```

Comment:
this and the package-lock changes worry me. Maybe you can make a new branch from master and make just the two changes below.